src/libs/ReportUtils.ts

```diff
@@ -5225,7 +5217,12 @@ function getIOUReportActionDisplayMessage(reportAction: OnyxEntry<ReportAction>)
         return Localize.translateLocal(translationKey, {amount: formattedAmount, payer: ''});
     }
 
-    const transaction = getTransaction(originalMessage.IOUTransactionID ?? '');
+    // This log to server is temporary and needed to determine if there is a case we need the transaction param
+    // when we call getIOUReportActionDisplayMessage from ReportActionItemMessage
```

Comment:
I've created a followup issue so that this will get cleaned up. https://github.com/Expensify/App/issues/39408